nautobot/cloud/templates/cloud/cloudaccount_retrieve.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block extra_breadcrumbs %}
+                <li><a href="{% url 'cloud:cloudaccount_list' %}?provider={{ object.provider.pk }}">{{ object.provider }}</a></li>
+{% endblock extra_breadcrumbs %}
+
+{% block content_left_page %}
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <strong>Cloud Account</strong>
+            </div>
+            <table class="table table-hover panel-body attr-table">
+                <tr>
+                    <td>Name</td>
+                    <td>
+                        {{ object.name }}
+                    </td>
+                </tr>
+                <tr>
+                    <td>Provider</td>
+                    <td>{{ object.provider|hyperlinked_object }}</td>
+                </tr>
+                <tr>
+                    <td>Account Number</td>
+                    <td>
+                        <span class="hover_copy"><span id="copy_accountnumber">{{ object.account_number }}</span><button type="button" class="btn btn-inline btn-default hover_copy_button" data-clipboard-target="#copy_accountnumber">
```

Comment:
Does this "just work" or do we have to add anything to the JS to support it? I can't remember offhand.

Reply:
It just works with manual testing.